mkdocs.yml

```diff
@@ -3,105 +3,112 @@
 site_name: Algoritmekader [concept]
 site_description: Concept van het algoritmekader. Het doel daarvan is om overheden op praktische wijze te ondersteunen, zodat zij op een wettige en ethisch verantwoorde wijze algoritmes en AI-systemen gebruiken.
 watch:
-    - includes/
+      - includes/
```

Comment:
it seems you are using a different formatter then the one used by algoritmekader. please remove all the spaces and conform to algoritmekader style.